plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -278,44 +281,409 @@
 public class IcebergMetadata
         implements ConnectorMetadata
 {
+    public static final Set<String> UPDATABLE_TABLE_PROPERTIES = ImmutableSet.of(FILE_FORMAT_PROPERTY, FORMAT_VERSION_PROPERTY, PARTITIONING_PROPERTY, SORTED_BY_PROPERTY);
```

Comment:
I see there are multiple unrelated changes in this file. Could you please fix this?

Reply:
> I see there are multiple unrelated changes in this file. Could you please fix this?

fixed